src/masker.js

```diff
@@ -99,7 +99,15 @@ export function formatter(value, config) {
       valueIndex++
     } else {
       accumulator += maskChar
-      if (char === maskChar) valueIndex++ // user typed the same char
+      if (char === maskChar) {
```

Comment:
Given your examples, would it make sense to make this check be case insensitive?

Reply:
Yes. That's probably wise. I will update the PR.